qiskit/visualization/pulse/matplotlib.py

```diff
@@ -591,16 +599,16 @@ def draw(self, schedule, dt, interp_method, plot_range,
             tf = tf or 1
 
         # prepare waveform channels
-        (channels, output_channels,
+        (schedule_channels, output_channels,
          snapshot_channels) = self._build_channels(schedule, channels_to_plot, t0, tf)
 
         # count numbers of valid waveform
         n_valid_waveform, v_max = self._count_valid_waveforms(output_channels, scaling=scaling,
-                                                              channels_to_plot=channels_to_plot,
+                                                              channels_to_plot=channels,
```

Comment:
Can we update the name for this method, as well? Since it's a private method, we shouldn't have to deprecate `channels_to_plot`, we can just remove it.

Reply:
fixed @ https://github.com/Qiskit/qiskit-terra/pull/3394/commits/ea2e292b426508ec1b0d6440c32f069268be60f2